rest-api/src/main/java/gov/cms/qpp/conversion/api/services/QrdaServiceImpl.java

```diff
@@ -1,19 +1,25 @@
 package gov.cms.qpp.conversion.api.services;
 
-import gov.cms.qpp.conversion.Converter;
-import gov.cms.qpp.conversion.QrdaSource;
-import gov.cms.qpp.conversion.api.model.Constants;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
+import gov.cms.qpp.conversion.Converter;
+import gov.cms.qpp.conversion.QrdaSource;
+import gov.cms.qpp.conversion.api.model.Constants;
+import gov.cms.qpp.conversion.model.validation.MeasureConfigs;
+
 /**
  * Implementation of the QRDA-III to QPP conversion service
  */
 @Service
 public class QrdaServiceImpl implements QrdaService {
 	private static final Logger API_LOG = LoggerFactory.getLogger(Constants.API_LOG);
 
+	public QrdaServiceImpl() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public constructor by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)